src/tests/affinity.cpp

```diff
@@ -65,7 +65,7 @@ set_thread_affinity(std::thread::native_handle_type thread,
             (thread_policy_t)&policy, 1);
 #elif defined(_WIN32) || defined(__CYGWIN__)
     rc = SetThreadAffinityMask(reinterpret_cast<HANDLE>(thread), 1ULL << cpuid) == 0 ? -2 : 0;
-#else
+#elif !defined(__OpenBSD__)
```

Comment:
What's wrong with this on OpenBSD?

Reply:
Nvm, I missed your comment.

Reply:
This should actually be rewritten for OpenBSD: https://man.openbsd.org/NetBSD-7.0.1/affinity.3
Struct name is `cpuset_t` there.

Reply:
I may be mistaken but you are referring to a man page of NetBSD.
I can't find either one of these in the OpenBSD sources: `cpu_set_t`, `cpuset_t`, `CPU_ZERO`, `CPU_SET`.

I think there is no thread affinity for OpenBSD: https://man.openbsd.org/pthreads.3

Reply:
Yes, you're right and it seems to be a design decision for OpenBSD.